monai/data/wsi_reader.py

```diff
@@ -85,6 +85,18 @@ def get_level_count(self, wsi) -> int:
         """
         raise NotImplementedError(f"Subclass {self.__class__.__name__} must implement this method.")
 
+    @abstractmethod
+    def get_downsample(self, wsi, level: int) -> float:
```

Comment:
Maybe update the function name to make it more clear: `get_downsample_ratio()`?

Thanks.